### PR TITLE
updated with budget values

### PR DESCRIPTION
**Pull request by Scient025**
budget

### CODE REVIEW

File: costing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Total Costs
+
+1. **Cost considering the 1st CAN shield and lower-end solder kit**:  
+   **6700 - 7000**
+   
+2. **Cost considering the higher-end CAN shield and solder kit**:  
+   **10,000 - 11,000**
+
+
```
Comment (TakshDhabalia) [inline costing.md:33]: Add Voltmetre costing with link

**Review by TakshDhabalia — CHANGES_REQUESTED** [review]
Please look into it

**Review by IRIS-MITWPU — COMMENTED** [review]
Please look into this as well

**Comment by Scient025** [comment]
updated with requested changes

File: costing.md [hunk 2]
```diff
@@ -1,37 +1,44 @@
...
 
 ## CAN Shield
-- **CAN shield (1st option)**: 200
-- **CAN shield (higher-end option)**: 700
+- **CAN shield (1st option)**: 200 : https://zbotic.in/product/cjmcu-2551-mcp2551-can-protocol-controller-high-speed-interface-module/?gad_source=1
+- **CAN shield (higher-end option)**: 700 : https://robu.in/product/mcp2515-can-bus-shield-board-spi-interface-9-pins-standard-sub-d-connector-expansion-module/
 
 ## Miscellaneous
-- **Jumper wires**: 300
+- **Jumper wires**: 300 : https://robocraze.com/products/jumper-wire-set-m2m-m2f-f2f-40-pcs-each?variant=40192573636761&currency=INR&campaignid=21586486032&adgroupid=&keyword=&device=c&gad_source=1
```
Comment (IRIS-MITWPU) [inline costing.md:17]: try using robou or zbotic if you can , otherwise we can use local vendor quotes

File: costing.md [hunk 3]
```diff
@@ -1,37 +1,44 @@
...
+- **Website domain**: 1000 : https://www.godaddy.com/en-in/domainsearch/find?domainToCheck=iris-mitwpu.edu
+- **Website backend hosting**: 600 : https://railway.app/pricing
 
 ## Essential Hardware Components
-- **Linear Actuator**: 3300
-- **STM32 board**: 1400
-- **USB to OBD**: 250
+- **Linear Actuator**: 3300 : https://robu.in/product/linear-actuator-stroke-length-100mm7mm-s1500n12v/
+- **STM32 board**: 1400 : https://robu.in/product/stmicroelectronics-development-board-stm32-nucleo-64-mcu-arduino-uno-v3-connectivity-flexible-power-supply/
+- **USB to OBD**: 250 : https://robu.in/product/mini-v2-1-elm327-obd2-bluetooth-interface-auto-car-scanner/
```
Comment (IRIS-MITWPU) [inline costing.md:10]: This is wireless , we require a wired board config wire .
try adding that , local source can be done .

File: costing.md [hunk 4]
```diff
@@ -1,37 +1,44 @@
...
-   **6700 - 7000**
+1. **Cost considering the 1st CAN shield, robou voltmeter and lower-end solder kit**:  
+   **7300 - 7500**
    
 2. **Cost considering the higher-end CAN shield and solder kit**:  
-   **10,000 - 11,000**
+   **11,820 - 12,000**
 
 
 ## remaining budget considering the ceiling values in both
```
Comment (IRIS-MITWPU) [inline costing.md:42]: add single and multi stranded wires , tape etc in 2k Misc. costs

**Review by IRIS-MITWPU — CHANGES_REQUESTED** [review]

File: costing.md (hunk 4 above)
Comment (IRIS-MITWPU) [reply costing.md:42]: also add STM connector wires as well with prod link

**Pull request merged**